pkg/kubelet/apis/config/types.go

```diff
@@ -170,6 +170,8 @@ type KubeletConfiguration struct {
 	// frequency and post node status immediately if any change is detected. It is
 	// only used when node lease feature is enabled.
 	NodeStatusReportFrequency metav1.Duration
+	// nodeLeaseRenewFrequency is the frequency that kubelet renews its corresponding Lease.
```

Comment:
when NodeLease feature is enabled:
https://github.com/kubernetes/kubernetes/blob/323356fda06cbac0b74c35f522f17c3bb9d580ca/pkg/features/kube_features.go#L544